README.md

```diff
@@ -0,0 +1,19 @@
+# GitPinner 
+
+Welcome to GitPinner
+
+### Instructions and use
+
+- Clone and cd into file
+- Run `npm install`
+- Run `npm run dev`
+
+### Wins
+- API runs
+- Cards display with data
+- User input form works
+
+### Challenges 
+- Not deployed yet
+
+Semhar & Greta

```

Comment:
Good Readme covering all the main aspects - maybe add a bugs section next time? (Although doesn't seem like you had any by the looks of it!)